src/pages/PrivateRoute.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { authenticate } from '../lib/auth';
+
+export const PrivateRoute = ({ component: Component, render, ...rest }) => (
```

Comment:
Let us put this in `lib` folder and name it `AuthenticateRoute`

Reply:
ok thanks for the feedback